### DISCUSSION TITLE
Mage - Vitriol

### DISCUSSION THREAD

**Discussion by Easygamer01s** [Crownicles/Crownicles#1936] [Nouvelles idées]
Bonjour à tous, 

Une suggestion concernant le mage dans les fights.. Vu que les effets se marchent un peu sur les pieds, je propose la transformation suivante :
- Ajout d'une nouvelle altération : "Vitriolé"
- Cette altération apparait lorsque la victime est déjà brulée ou empoisonnée et que le mage fait l'autre attaque..

A mon sens, il s'agirait de modifier les attaques de BurnAttack et PoisonAttack pour importer l'altération de l'opposant et la transformée en cas de succès de l'attaque. Ainsi que d'ajouter une nouvelle altération.

Après on peut imaginer ce que l'on veut sur les conséquences de cette nouvelle altération =D

On peut imaginer aussi donner un sens en ne modifiant qu'une des deux attaques pour que le joueur doive d'abord entrer Poison, puis Burn.. 

ça donnerait des combots à rentrer dans un ordre précis.. voir si ça s'aligne avec le souffle.. Quid de CursedAttack ?

Cordialement,
Easy

**Comment by BastLast** [comment]
J'aime bien cette idée, j'y penserai à l'occasion

**Comment by Bombafox** [comment]
Je suis complètement pour des attaques + intéressante
J'ai casiment arrêté de fight tellement c'est peu intéressant les combats pour moi
